### ISSUE TITLE
Mid-2024 redesign of landing / home page and static pages

### ISSUE THREAD

**Issue by fancyham** [hackforla/food-oasis#2195]
### Context

We're looking to make the home page more welcoming and useful : showing information about ourselves, what to expect, what's different about us, help make people feel good and hopeful, and understand more about us (for example, that we're a mission-based volunteer org)

To do this, we're planning on expanding the homepage and putting more content on it, as well as linking out to new content. 

### How is it now?
<img src="https://github.com/user-attachments/assets/b102959e-b09c-4469-a38e-ace651705581" width="400" />

Right now, the landing page is a greyed-out map with a single search field on top.

Problems with this: 
- It seems (based on Clarity click heat maps) that some people think that the map live and try to click/tap on it
- Missed opportunity to introduce ourselves and what makes us different from other free-food search directories
- Kinda boring and generic! (I designed it and it's totally true)
 
### Proposed new feature
With this new landing page, we want users to feel more in-control, hopeful, and informed than before they arrived.  

The directory is a big part of what we do, but we can also reassure them by helping people connect to other resources, explain confusing terminology, or to help them feel comfortable getting free food.  We can also be honest about any limitations of our site.

Solving this GitHub issue will involve sussing out what our communications priorities are and gut-checking it internally.  Please do lots of reviews with folks along the way!  We are a unique organization, so how we solve it might be different from anything else out there.

### Discussion
- Here are some sketches of what could be to get the ideas flowing:
(Figma) https://www.figma.com/design/9hPmNSsyaH2VMGNz5mffmI/FOLA-Design-%232?node-id=8032-10466&t=Mbo9rOXnAezRPe7e-4
- What are some other possible solutions?
- What are some challenges?

### To-do (click the … > edit button in the top right to edit these as necessary)
- [x] Design layout sketch [(link to figma)](https://www.figma.com/design/9hPmNSsyaH2VMGNz5mffmI/FOLA-Design-%232?node-id=5540-1532&t=74zMkoM1lcAZQk2h-4)
![image](https://github.com/user-attachments/assets/5ceba9b3-6c88-41d5-b7e8-057785064bfa)
- [x] Draft content for landing page -- text, photos/illos, connections between pages (google doc, probably)
- [ ] Mock up landing page using this, then run it by people and get their feedback
- [ ] Revision cycles for content and design
- [ ] Finalize design and content once everyone is happy -- it meets FOLA's goals
- [ ] Bundle everything up for devs
- [ ] Development / build it
- [ ] QA on dev site

### References:
- Older versions of foodoasis.la site were also quite vibrant and sometimes fiery, so that's an option for us as well.
Vibrant landing page - https://web.archive.org/web/20200520140138/https://foodoasis.la/
- Old about us pages 
  - https://web.archive.org/web/20200512164635/https://foodoasis.la/about/
  - https://web.archive.org/web/20220328124205/https://foodoasis.la/about
- Old team page - https://web.archive.org/web/20200527190646/https://foodoasis.la/team/

**Comment by Shienny1** [comment]
All, 

https://docs.google.com/document/d/1E5WKAa5GS8woGmoSOwrUJlrRoKOHfNQ5/edit?usp=sharing&ouid=100212516031199906779&rtpof=true&sd=true

Please take a look at this document for DRAFT content text for:
Updated 17 September:
-**Landing Page** -- pending neighborhood council data
-**Partner Page** - this page is to provide information on how organizations partner with FoodOasis.
-**Volunteer page** - this page is to show "volunteer listing" specifically for FOLA student-volunteer Coordinator and also provide a link to Hack for LA.

Updated 17 August:
- **About Page / Mission Statement**
- **Our Team**
- **For Donors**

**Comment by fancyham** [comment]
Reviewed the Google Doc draft manuscript above and left some notes and ideas :)

**Comment by Shienny1** [comment]
@JohnHaoHuang FYI - This task has been de-prioritized because we are fixing bugs for the map first.

**Comment by samhedlund** [comment]
**As I pick up work on this in 2026 I’m starting with these questions:** 
_I'll discuss briefly with dev and design teams tonight and schedule time to chat more with @fancyham and the leadership meeting as needed._

**1. Do our checklist steps still make sense? What's next?**
- I recommend we split off any related non-landing-page work on static pages off into separate ticket(s).

**2. What should we treat as our new starting draft for landing-page language?** I’m leaning toward the version in Figma because the Docs version seems too wordy to be mobile-friendly and uses some concepts I'd like to know more about like “start by identifying your Neighborhood Council.”

**3. If we want to revisit A1 versus other options on Figma...What kind of food and/or map imagery combines accuracy,completeness, and attractiveness? **

**Comment by fancyham** [comment]
Hi Sam,

Re: #1 the checklist items: 

Since you're picking this up, I think it makes sense to take a few steps back in the checklist.  For example, the layout part of issue is a bit premature (and actually, the designer was trying to show a possible template as a sketch, but that's where he stopped)

I think the goal still goes to: How can we describe ourselves better on our site to our various audiences, and what possible things might we consider for marketing ourselves. 

Also check out these two issues that touch a bit on what we've been talking about together.

Also, FYI, these other issues might be a bit too zoomed-out or looking at the wrong aspect, but here's what we had in our issues database:
- #2170 (too broad — it's an epic that would need to be broken into smaller pieces)
- #1620 (marketing, so probably not really the web site's content)

**Comment by fancyham** [comment]
Here's a rough chronology — you might be able to see more on archive.org for things like team, FAQ, etc.

- 2017: https://web.archive.org/web/20170503224116/https://foodoasis.la/about/
- 2019: https://web.archive.org/web/20191020153350/https://foodoasis.la/about/
  - Also: here's a page that didn't change since 2019: https://mckinney.foodoasis.net/about (before COVID!)
- 2022: https://web.archive.org/web/20220328124205/https://foodoasis.la/about/ (mentions a lot of COVID)
- 2023 onward: An issue #1365 was to update the text which had a lot of COVID mentions... but that update also removed a ton of personality.  Unfortunately, archive.org didn't/can't capture the pages from this point on: #1365
  - Here's our current text: https://foodoasis.la/about which hasn't changed since that #1365

**Comment by fancyham** [comment]
Hey Sam, we tried to recap what we talked about in our Leads/PM meeting:

- [ ] Questions for team
  - [ ] Bear in mind that I’m starting with web content but also thinking ahead to marketing campaigns.
  - [ ] Questions (time permitting, will discuss in meeting):
    - [ ] How would you describe the top priorities for Food Oasis? E.g.:
      - [ ]  Help more food-seekers, better, as measured by site usage and satisfaction? : Food seekers, people helping food seekers, making sure folks get the right info and know where we're coming from, trusting us, and leaving hopeful.  Mentioned we've not been concentring on volunteers.  Embedding opportunities, not sure what's happening on that front but it used to be a thing.  We don't expect food-seekers to return, but helpers will come back often.  Relates to how our metrics differ from other sites.  Talk to Shienny and Bonnie re: outreach to neighborhood councils.  Partners... we haven't really been in touch with most of them in years.
      - [ ] Getting more of the right volunteers? : We could make it easier than it is now.
      - [ ] Making it easier to submit corrections? 
    - [ ] Any preferences or guardrails on messaging, tone, mission statement (will discuss in meeting) : We talked about missing the activist vibe, missing the people.  Adding a landing page describing what we're doing.
    - [ ] Any general feedback on the current site content and design? (Will discuss problems and solutions in meeting) :  Bryan said: kinda boring now, especially compared to before.
- [ ] Any ideas or preferences on zero-cost promotional strategies? (Will discuss options in meeting) : Use existing networks of helpers — teachers, social worker word of mouth, opportunity for press-blurbs that are easy to copy and paste by journalists.  1 and 3 paragraph, photos.  Media kit.  We have social media channels exist but are dead.

@ExperimentsInHonesty and @Shienny1 may add their own notes probably here
